### ISSUE TITLE
Option to return imosToolbox version string

### ISSUE THREAD

**Issue by sspagnol** [aodn/imos-toolbox#437]
If batch auto processing I want to set up some output directories before calling imosToolbox, can we have an option to call imosToolbox and get the toolbox version back, eg 

toolboxVersion = imosToolbox(true,'-version')

and toolboxVersion is as set in imosToolbox.

**Comment by ggalibert** [comment]
Yes, can do. In the next release we can have the following function in the Util directory:

```Matlab
function toolboxVersion = getToolboxVersion()
%GETTOOLBOXVERSION Retrieves the current version of the toolbox and the
%architecture used

toolboxVersion = ['2.5.27 - ' computer];

end
```

**Comment by ggalibert** [comment]
Ok, will change imosToolbox.m so that calling

```Matlab
imosToolbox('version')
```

dispalys the toolbox version.

**Comment by ggalibert** [comment]
Fixed in v2.5.27 by https://github.com/aodn/imos-toolbox/commit/e0f5ab7a19c45be6155ca10cb25be89859bf1b07

**Comment by ggalibert** [comment]
Since v2.5.27, you can retrieve the version number programmatically in a string by doing:
```Matlab
version = evalc('imosToolbox(''version'')')
```
